### PR TITLE
Port quick start to dune syntax

### PR DESCRIPTION
**Pull request by rgrinberg**
Signed-off-by: Rudi Grinberg <rudi.grinberg@gmail.com>

### CODE REVIEW

File: doc/quick-start.rst [hunk 1]
```diff
@@ -65,17 +62,14 @@ The executable will be built as ``_build/default/hello_world.exe``
 Building a hello world program using Core and Jane Street PPXs
 ==============================================================
 
-Write this jbuild:
+Write this dune:
```
Comment (emillon) [inline doc/quick-start.rst]: "This `dune` file" maybe?

Comment (emillon) [reply doc/quick-start.rst]: (obviously this applies to all instances - not sure "a dune" is a great name)

**Comment by rgrinberg** [comment]
@emillon It's now dune file everywhere.

**Comment by emillon** [comment]
Thanks! :+1:

**Pull request merged**